sklearn/utils/estimator_checks.py

```diff
@@ -2352,8 +2352,8 @@ def check_decision_proba_consistency(name, estimator_orig):
             hasattr(estimator, "predict_proba")):
 
         estimator.fit(X, y)
-        a = estimator.predict_proba(X_test)[:, 1]
-        b = estimator.decision_function(X_test)
+        a = estimator.predict_proba(X_test)[:, 1].round(decimals=10)
```

Comment:
That's not great :-/ Can we avoid that? Basically you're saying after the "fix" the estimator is not consistent any more because of floating point issues?

Reply:
because scipy's `expit` isn't precise enough. But I feel like 1e-10 is a decent precision. I had issues with this test before and IMO it's too strict.

Reply:
Agreed it's fine.

Reply:
Should we be rounding as a non-copying operation? Is this copying?

Reply:
It's copying.

I can't tell from the docs (nor the code) whether it copies if we pass `out=`.

I don't think that's important though?

Reply:
Using `out=` will not copy if it's into the same array. 
I forgot this was in estimator checks. Yes, not important.